Code/Framework/AzCore/Platform/Android/AzCore/AzCore_Traits_Android.h

```diff
@@ -121,3 +121,8 @@
 #define AZ_TRAIT_LEGACY_CRYCOMMON_USE_WINDOWS_STUBS 1
 #define AZ_TRAIT_LEGACY_CRYPAK_UNIX_LIKE_FILE_SYSTEM 1
 #define AZ_TRAIT_LEGACY_CRYSYSTEM_DEFINE_DLLMAIN 0
+
+#if defined (CARBONATED)
+// Carbonated traits ...
+#define AZ_TRAIT_USE_POLL_INSTEAD_SELECT_FOR_SOCKET_POLLING 1
+#endif
```

Comment:
Suggest a more concise name, starting with the system and then the usage:

e.g
`#define AZ_TRAIT_SOCKETS_USE_POLL 1`

and that then provides a space for additional socket traits, e.g.
`#define AZ_TRAIT_SOCKETS_COMPRESSION 0`
`#define AZ_TRAIT_SOCKETS_PANTS 0`

.. etc.